app/services/spree/retail/shopify/refunder.rb

```diff
@@ -1,14 +1,16 @@
 module Spree::Retail
   module Shopify
     class Refunder
-      def initialize(credited_money:, transaction_id:, transaction_interface: nil, refunder_interface: nil, can_issue_refund_policy_klass: nil, **options)
-        @refund_reason = options[:reason]
-        @order_id = options[:order_id]
+      def initialize(credited_money:, transaction:, refunder_interface: nil, can_issue_refund_policy_klass: nil, **options)
         @credited_money = BigDecimal.new(credited_money)
-        @transaction_interface = transaction_interface || default_transaction_interface
+        @transaction = transaction
+        @refund_reason = options[:reason]
+
+        # FIXME: HOOOOOOORJ This is abominable
+        @order_id = transaction.prefix_options[:order_id]
```

Comment:
This is the only really gnarly bit of implementation here. `transaction` _claims_ it responds to `.order_id`, but its response is always nil. I'm tempted to monkeypatch `ShopifyAPI::Transaction` to fix its interface. Thoughts?